### PR TITLE
f-offers@v0.4.0 - NoResults component

### PR DESCRIPTION
**Pull request by matthewhardern**
This PR contains the NoResults component code, which is part of the offers page. Not doing UI review checks at this stage, just trying to get all the components built first, then later doing a review check on the whole `f-offers`

- [x] Unit tests have been [created|updated]

### CODE REVIEW

File: packages/components/organisms/f-offers/src/components/NoResults.vue [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<template>
+    <div
+        :class="$style['c-noResults-wrapper']"
+    >
+        <media-element
+            :title="$t('noResults.title')"
+            :text="$t('noResults.subtitle')"
+            image-url="https://d30v2pzvrfyzpo.cloudfront.net/a/hw/img/decoration/oi_no-results-image.svg"
```
Comment (minrwhite) [inline packages/components/organisms/f-offers/src/components/NoResults.vue]: I'm assuming this is in lieu of an agreed cloudinary upload location?

File: packages/components/organisms/f-offers/src/components/_tests/NoResults.test.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+const mockLocale = 'en-GB';
+
+describe('NoResults.vue', () => {
+    let wrapper;
+    beforeEach(() => {
+        // Arrange
+        wrapper = mount(NoResults, {
+            mocks: {
+                $t: key => key.split('.').reduce((acc, current) => acc[current], tenantConfigs[mockLocale].messages)
```
Comment (minrwhite) [inline packages/components/organisms/f-offers/src/components/_tests/NoResults.test.js]: _Nice_

File: packages/components/organisms/f-offers/src/components/NoResults.vue (hunk 1 above)
Comment (matthewhardern) [reply packages/components/organisms/f-offers/src/components/NoResults.vue]: Yea I am still waiting on a reply to my message in help-content, will chase up again today

File: packages/components/organisms/f-offers/src/components/Offers.vue [hunk 3]
```diff
@@ -2,36 +2,30 @@
     <div
         :class="$style['c-offers']"
         data-test-id="offers">
-        <offers-search-box />
+        <no-results />
```
Comment (zlatinivanov-je) [inline packages/components/organisms/f-offers/src/components/Offers.vue]: If feels little confusing here as the file name suggests Offers, but those seem to have been replaced.

File: packages/components/organisms/f-offers/package.json [hunk 4]
```diff
@@ -58,6 +59,8 @@
     "@vue/test-utils": "1.0.3",
     "node-sass-magic-importer": "5.3.2",
     "js-cookie": "2.2.1",
-    "@justeat/f-trak": "0.7.1"
+    "@justeat/f-trak": "0.7.1",
+    "@justeat/f-media-element": "0.3.0",
+    "@justeat/f-button": "1.5.0"
```
Comment (minrwhite) [inline packages/components/organisms/f-offers/package.json]: Any reason not to use the latest `f-button` - `1.8.0` at time of writing?

Comment (matthewhardern) [reply packages/components/organisms/f-offers/package.json]: No sorry, its what my webstorm suggested, will make the change

**Review by zlatinivanov-je — COMMENTED** [review]
One small comment, would it make sense to use `noOffersFound/Available` or similar. 
Maybe I am missing on the context here, please feel free to correct me. 
Other than that happy to approve

File: packages/components/organisms/f-offers/src/components/Offers.vue (hunk 3 above)
Comment (matthewhardern) [reply packages/components/organisms/f-offers/src/components/Offers.vue]: No results is a component that is shown on the offers page (where the cards would appear) at all times. So I agree the `NoOffersFound` maybe a better name, thanks

File: packages/components/organisms/f-offers/package.json (hunk 4 above)
Comment (matthewhardern) [reply packages/components/organisms/f-offers/package.json]: 1.7.0 is only option

File: packages/components/organisms/f-offers/src/components/_tests/NoResults.test.js (hunk 2 above)
Comment (matthewhardern) [reply packages/components/organisms/f-offers/src/components/_tests/NoResults.test.js]: thanks

File: packages/components/organisms/f-offers/src/components/Offers.vue (hunk 3 above)
Comment (matthewhardern) [reply packages/components/organisms/f-offers/src/components/Offers.vue]: changed to your suggestion

**Review by minrwhite — APPROVED** [review]

**Review by JoshuaNg2332 — APPROVED** [review]

**Pull request merged**